### PR TITLE
DE47831 - Fix height issue

### PR DESCRIPTION
**Pull request by devpow112**
This also get's rid of a weird flicker in the hover styles that was occurring due to a brief delay between the parent rendering and the edit element rendering.

**Before**
before

**After**
after

### CODE REVIEW

File: src/components/rubric-editable-score.js [hunk 1]
```diff
@@ -121,8 +123,12 @@ export class RubricEditableScore extends composeMixins(
 					background-color: var(--d2l-color-celestine-plus-2);
 				}
 
-				:host(.assessable:not([editor-styling])) #input-container:focus,
-				:host(.assessable:not([editor-styling])) #input-container:hover {
+				.editing {
+					border-color: transparent !important;
+				}
+
+				:host(.assessable:not([editor-styling])) :not(.editing) #input-container:focus,
```
Comment (devpow112) [inline src/components/rubric-editable-score.js:130]: This is starting to get a bit ridiculous. Might look at optimizing this in the future.

**Comment by devpow112** [comment]
VDiff tests should fail as this was broken in the vdiff tests for assessed rubrics.

File: src/components/rubric-editable-score.js [hunk 2]
```diff
@@ -738,18 +750,35 @@ export class RubricEditableScore extends composeMixins(
...
-				${this._scoreOverridden && !this.readOnly ? html`
-					<div class="d2l-offscreen">${this._localizeStarLabel()}</div>
-				` : null}
+				${this._renderScoreOverriddenOffscreen()}
+			</div>
+		`;
+	}
+
+	_renderScoreOverriddenOffscreen() {
+		if (!this._scoreOverridden || this.readOnly) {
```
Comment (devpow112) [inline src/components/rubric-editable-score.js:763]: I think this is the opposite of `this._scoreOverridden && !this.readOnly` logically. We used to use `this._scoreOverridden && !this.readOnly` to determine if we should render the override offscreen. Now we early exist and don't render.

**Comment by devpow112** [comment]
VDiff look ok to me. There is some resizing occurring because before the height of the edit score component could vary depending on if the text was unaligned. Now it's consistent everywhere.

**Review by mpharoah-d2l — APPROVED** [review]

**Comment by devpow112** [comment]
Unit tests are gonna fail. Installing dependencies got stuck for some reason. Maybe a good time to look at doing #1762.

**Review by MitchelPaulin — APPROVED** [review]

**Comment by devpow112** [comment]
Safari is being flakier then usual.

**Pull request merged**